filebeat/harvester/harvester.go

```diff
@@ -17,12 +17,15 @@ import (
 	"fmt"
 	"regexp"
 
+	"github.com/satori/go.uuid"
+
 	"github.com/elastic/beats/filebeat/config"
 	"github.com/elastic/beats/filebeat/harvester/encoding"
 	"github.com/elastic/beats/filebeat/input"
 )
 
 type Harvester struct {
+	Id                 uuid.UUID
```

Comment:
If we write the Id to logs and to stats, it is somehow persisted. Assuming logs and stats are stored in ES, it will make finding a specific harvester easier if we go with UUID I think as only one filter has to be applied. WDYT?